meta-resin-common/recipes-connectivity/networkmanager/files/NetworkManager.conf

```diff
@@ -4,4 +4,4 @@ plugins=keyfile
 autoconnect-retries-default=0
```

Comment:
The missing Changelog-entry is on purpose?

Reply:
It defaults to the commit subject and since I find it descriptive enought, I left it as it is.